src/Native/System.Native/pal_process.cpp

```diff
@@ -323,3 +329,74 @@ extern "C" int32_t WTermSig(int32_t status)
 {
     return WTERMSIG(status);
 }
+
+extern "C" int64_t PathConf(const char* path, PathConfName name)
+{
+    int32_t confValue = -1;
+    switch (name)
+    {
+        case PathConfName::PAL_PC_LINK_MAX:
+            confValue = _PC_LINK_MAX;
+            break;
+        case PathConfName::PAL_PC_MAX_CANON:
+            confValue = _PC_MAX_CANON;
+            break;
+        case PathConfName::PAL_PC_MAX_INPUT:
+            confValue = _PC_MAX_INPUT;
+            break;
+        case PathConfName::PAL_PC_NAME_MAX:
+            confValue = _PC_NAME_MAX;
+            break;
+        case PathConfName::PAL_PC_PATH_MAX:
+            confValue = _PC_PATH_MAX;
+            break;
+        case PathConfName::PAL_PC_PIPE_BUF:
+            confValue = _PC_PIPE_BUF;
+            break;
+        case PathConfName::PAL_PC_CHOWN_RESTRICTED:
+            confValue = _PC_CHOWN_RESTRICTED;
+            break;
+        case PathConfName::PAL_PC_NO_TRUNC:
+            confValue = _PC_NO_TRUNC;
+            break;
+        case PathConfName::PAL_PC_VDISABLE:
+            confValue = _PC_VDISABLE;
+            break;
+    }
+
+    if (confValue == -1)
+    {
+        assert(false && "Unknown PathConfName");
+        return EINVAL;
```

Comment:
This should return -1 and set errno to EINVAL to mimic the semantics of underlying API.